setup.py

```diff
@@ -20,8 +19,6 @@
     include_package_data=True,
     python_requires=">=3.6",
     install_requires=[
-        "setuptools",
```

Comment:
sure  this is a good idea? It will be there anyway. And `pkg_resources` (part of setuptools) is needed. Not sure though, if here.

Reply:
You are right. `pkg_resources` is used. I will return `setuptools`.

Reply:
Done.